modules/test/playwright/helpers/ApiBuilderHelper.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ApiHelpers} from './ApiHelpers';
+
+export class ApiBuilderHelper {
+	readonly apiHelpers: ApiHelpers;
+	readonly basePath: string;
+
+	constructor(apiHelpers: ApiHelpers) {
+		this.apiHelpers = apiHelpers;
+		this.basePath = 'headless-builder';
+	}
+
+	async getEndpointPage(endpointURI: string) {
+		return this.apiHelpers.getResponse(
+			`${this.apiHelpers.baseUrl}${endpointURI}`
+		);
+	}
+
+	async postApiApplication(data: DataObject) {
+		return this.apiHelpers.post(
+			`${this.apiHelpers.baseUrl}${this.basePath}/applications`,
```

Comment:
It is a bit confusing having some paths ending with slash `/` (this.apiHelpers.baseUrl) and some are not (`this.basePath`). Could we unify them?